src/views/v2/tickets/item/content/description.php

```diff
@@ -38,5 +38,5 @@
 	id="<?php echo esc_attr( $ticket_details_id ); ?>"
 	class="tribe-common-b2 tribe-common-b3--min-medium tribe-tickets__item__details__content"
 >
-	<?php echo $ticket->description; ?>
+	<?php echo esc_html_x( $ticket->description, 'Ticket Description in Ticket Form', 'event-tickets' ); ?>
```

Comment:
:no_entry_sign: The $text arg must be a single string literal, not "$ticket->description".

<sub>hash: 7aec2480fd2c7d937e1a795d5b66de5b</sub>

Reply:
```suggestion
	<?php echo esc_html( $ticket->description ); ?>
```